lib/constants.js

```diff
@@ -6,6 +6,7 @@ export const ALGORITHM = 'ECDSA';
 // Determines whether key pair is extractable
 export const EXTRACTABLE = true;
 // ECDSA curve P-256 type
+export const ECDSA_2019_SECP_256_K1_KEY_TYPE = 'EcdsaSecp256k1VerificationKey2019';
```

Comment:
Hmm... for consistency with the other constants / values:

```suggestion
export const ECDSA_2019_SECP_256K1_KEY_TYPE = 'EcdsaSecp256k1VerificationKey2019';
```